fastexcel-writer/src/test/java/org/dhatim/fastexcel/WriteFileTest.java

```diff
@@ -0,0 +1,25 @@
+package org.dhatim.fastexcel;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.math.BigDecimal;
+
+public class WriteFileTest {
+
+  @Test
+  void testWrite() throws IOException {
+    try (FileOutputStream os = new FileOutputStream("/home/mathieu/testFormulatWithoutValue.xlsx")) {
```

Comment:
I suggest to remove path to your home and change it with target maven dir or /tmp dir

Reply:
removed